### DISCUSSION TITLE
Automatically detect onion url via `Onion-Location` header

### DISCUSSION THREAD

**Discussion by ononoki1** [searxng/searx-instances#194] [Ideas]
Usually, if an instance has onion entry, it will send `Onion-Location` header in every clearnet request. We can use that to automatically detect its onion url and display that on searx.space.

In this way, searx-instances reviewers do not need to deal with onion url add/delete/edit issues. If an instance maintainer adds/deletes/edits its onion url, he/she only needs to modify the `Onion-Location` header and searx.space will automatically update it.

**Comment by unixfox** [comment]
Good idea, but that will only apply to the instances that are running the same server on the clearnet and on the tor network.

Do we deny any request for new onion URLs where the instance exist on the clearnet and on the tor network then? Or we kindly ask to add this parameter instead?

Btw for new feature requests on searx.space it's there: https://github.com/searxng/searx-space/issues
